subt/main.py

```diff
@@ -497,6 +498,7 @@ def on_pose3d(self, timestamp, data):
             # we cannot align global coordinates if offset is not known
             return
         xyz, rot = data
+        self.orientation = rot  # quaternion
```

Comment:
What is the plan for robots/configurations that still use on_pose2d?

Reply:
it is even worse - what to do with robots, which use old artifact reporter -> it will crash now. I suggest to add it to #640 and review it a week later